ansible/site.yml

```diff
@@ -166,3 +166,18 @@
       when:
         - enable_orchestration == true
   tags: orchestration
+
+- name: Install SIM
+  hosts: controllers
+  remote_user: root
+  vars_files:
+    - group_vars/common.yml
+    - group_vars/sim.yml
```

Comment:
Can we switch on only SIM installation ? is there an option in common.yml to discard everything?

Reply:
As mentioned in PR, 

Change file ansible/group_vars/sim.yml, "enable_sim: true"
Run following command to install only SIM: "installer/ansible# ansible-playbook site.yml -i local.hosts -vv --tags sim"